lib/dal/src/diagram.rs

```diff
@@ -296,6 +130,25 @@ impl SummaryDiagramEdge {
             from_base_change_set: false,
         })
     }
+
+    pub fn assemble_outgoing(
+        outgoing_connection: OutgoingConnection,
+        from_component: &Component,
+        to_component: &Component,
+        change_status: ChangeStatus,
+    ) -> ComponentResult<Self> {
+        Ok(SummaryDiagramEdge {
+            from_component_id: outgoing_connection.from_component_id,
```

Comment:
Are these from & to "correct"? I don't have to flip them?

Reply:
yeah those look right to me! you can also just pull the to and from off the outgoing_connection instead of also passing them in